libs/vm-modules/tests/unit/big_number_wrapper_tests.cpp

```diff
@@ -189,66 +189,211 @@ TEST_F(UInt256Tests, uint256_raw_increase)
 TEST_F(UInt256Tests, uint256_comparisons)
 {
   static constexpr char const *TEXT = R"(
-    function main() : Bool
-        var ok : Bool = true;
-        var uint64_max = 18446744073709551615u64;
+    function main()
+      var uint64_max = 18446744073709551615u64;
+      var smaller = UInt256(uint64_max);
+      var bigger = UInt256(uint64_max);
+      bigger.increase();
+
+      assert(smaller < bigger, "1<2 is false!");
+      assert((smaller > bigger) == false, "1>2 is true!");
+      assert(smaller != bigger, "1!=2 is false!");
+      assert((smaller == bigger) == false, "1==2 is true!");
+    endfunction
+  )";
 
-        var smaller = UInt256(uint64_max);
-        var bigger = UInt256(uint64_max);
-        bigger.increase();
+  ASSERT_TRUE(toolkit.Compile(TEXT));
+  EXPECT_TRUE(toolkit.Run());
+}
 
-        var gt : Bool = smaller > bigger;
-        ok = ok && !gt;
+TEST_F(UInt256Tests, uint256_assignment)
+{
+  // NOTE! The current behavior for assignment of UInt256 is a reference-assignment,
```

Comment:
if this is still true you need to add TODO(issue number) here

Reply:
There is no issue for this rework, afaik. @robertdickson told me that what I describe for UInt256 is an expected assignment behavior for Object value types, which are now not implemented.

Reply:
I have created https://fetchai.atlassian.net/browse/ML-258 for this rework, and for now just removed this comment.